egs/wsj/s5/steps/nnet3/chain/e2e/compute-biphone-stats.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
```

Comment:
please use underscores not dashes in the name, to match other script names.

Reply:
Done.